### PR TITLE
Created list from gdocs sheet

### PR DESCRIPTION
**Pull request by bayological**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,116 @@
...
+
+ 
+
+
+<details>
+  <summary><span style="font-size: 1.5em;">🎮 Games</span></summary>
+
+<br />
+
+| **Description** | **Link** |
```
Comment (ajsantander) [inline README.md:17]: Left column should be "Name" instead of "Description"?

File: README.md [hunk 2]
```diff
@@ -1 +1,116 @@
...
+| ----------- | ----------- |
+| Ethernauts | https://ethernaut.openzeppelin.com/ |
```
Comment (eternauta1337) [inline README.md]: I'd call this one "Ethernaut CTF"

File: README.md [hunk 3]
```diff
@@ -1 +1,116 @@
...
+<br />
+
+| **Description** | **Link** |
+| ----------- | ----------- |
+| The Complete Guide to Full Stack Ethereum Development      | https://dev.to/dabit3/the-complete-guide-to-full-stack-ethereum-development-3j13 |
+| How to become a DeFi Developer - Kerman Kohli| https://defiweekly.substack.com/p/how-to-become-a-defi-developer |
+| A Guide for Developers Interested in Learning Blockchain Development| https://www.linumlabs.com/articles/a-guide-for-developers-interested-in-learning-blockchain-development |
+| Taking undercollateralized loans for fun and for profit | https://samczsun.com/taking-undercollateralized-loans-for-fun-and-for-profit/ |
+| Resources for learning smart contract security | https://immunefi.com/learn/ |
+| Most common smart contract bugs of 2020 | https://medium.com/solidified/most-common-smart-contract-bugs-of-2020-c1edfe9340ac |
```
Comment (eternauta1337) [inline README.md:51]: Could you add a few of mine here 🙏 

This is a bit hardcore, but very useful
https://blog.openzeppelin.com/deconstructing-a-solidity-contract-part-i-introduction-832efd2d7737/

This one's nice for The Graph
https://theethernaut.substack.com/p/developer-superpowers-with-thegraph

File: README.md [hunk 4]
```diff
@@ -1 +1,116 @@
...
+</details>
+
+<details>
+  <summary><span style="font-size: 1.5em;">📣 Tweet Storms</span></summary>
+
+<br />
+
+| **Description** | **Link** |
+| ----------- | ----------- |
+| Ethernauts      | https://ethernaut.openzeppelin.com/       |
```
Comment (eternauta1337) [inline README.md]: ?

**Pull request merged**